### ISSUE TITLE
@graphql-toolkit dependencies use deprecated module

### ISSUE THREAD

**Issue by brettbullock** [Urigo/merge-graphql-schemas#384]
Affected version: v1.7.7

Merge-graphql-schemas uses 3 @graphql-toolkit dependencies:

- @graphql-toolkit/common
- @graphql-toolkit/file-loading
- @graphql-toolkit/schema-merging

In v1.7.7, the version sourced for these dependencies is v0.9.12, however this version seems to be using a deprecated graphql module.

My `package-lock.json`:
```
 "merge-graphql-schemas": {
      "version": "1.7.7",
      "resolved": "https://registry.npmjs.org/merge-graphql-schemas/-/merge-graphql-schemas-1.7.7.tgz",
      "integrity": "sha512-3mjFHXpsF+bnsnSy/FlZiWCcNo7AULuG8kEBE+YVX8qbzNrEPJVhSg7ANr5LjQZbGtSlTz7fa3Wqw0bfbQ5R5Q==",
      "requires": {
        "@graphql-toolkit/file-loading": "0.9.12",
        "@graphql-toolkit/schema-merging": "0.9.12",
        "tslib": "1.11.1"
      },
      "dependencies": {
        "@graphql-toolkit/common": {
          "version": "0.9.12",
          "resolved": "https://registry.npmjs.org/@graphql-toolkit/common/-/common-0.9.12.tgz",
          "integrity": "sha512-NjBVxeM1GB0bldiBm4UQoxKtbu/hjRfcqDEt1JPia+uXgFde3qTxtF5IZkworRFNCp+8KPSVrwcreQHSfFW63w==",
          "requires": {
            "aggregate-error": "3.0.1",
            "camel-case": "4.1.1",
            "graphql-tools-fork": "9.0.1",
            "lodash": "4.17.15"
          }
        },
        "@graphql-toolkit/file-loading": {
          "version": "0.9.12",
          "resolved": "https://registry.npmjs.org/@graphql-toolkit/file-loading/-/file-loading-0.9.12.tgz",
          "integrity": "sha512-xEmKkbWI5FXQARTN3dbPzshwcmduOJUvx1QGsfcuryVDdQJZynUEaPF7IxYaFpvQF7vwxGIRdff/jC/a5ypDrA==",
          "requires": {
            "globby": "11.0.0",
            "unixify": "1.0.0"
          }
        },
        "@graphql-toolkit/schema-merging": {
          "version": "0.9.12",
          "resolved": "https://registry.npmjs.org/@graphql-toolkit/schema-merging/-/schema-merging-0.9.12.tgz",
          "integrity": "sha512-ciqxLeMw7KPbJcq/xgnPbGyJGRO6bO1zQcdWCUSssyw8VDRHj5PFqEOAzT88eZQkEtg3qLN/wQEypeyFyNTHzw==",
          "requires": {
            "@graphql-toolkit/common": "0.9.12",
            "deepmerge": "4.2.2",
            "graphql-tools-fork": "9.0.1",
            "tslib": "1.11.1"
          }
        }
      }
    }
```

When trying to run my app I receive the following errors:
```
node_modules/merge-graphql-schemas/node_modules/@graphql-toolkit/common/fix-schema-ast.d.ts:2:25 - error TS2307: Cannot find module 'graphql/utilities/schemaPrinter'.

2 import { Options } from 'graphql/utilities/schemaPrinter';
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/merge-graphql-schemas/node_modules/@graphql-toolkit/common/parse-graphql-json.d.ts:4:25 - error TS2307: Cannot find module 'graphql/utilities/schemaPrinter'.

4 import { Options } from 'graphql/utilities/schemaPrinter';
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/merge-graphql-schemas/node_modules/@graphql-toolkit/common/print-schema-with-directives.d.ts:2:25 - error TS2307: Cannot find module 'graphql/utilities/schemaPrinter'.

2 import { Options } from 'graphql/utilities/schemaPrinter';
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 3 errors.
```

Navigating to the `graphql` node_module, the `graphql/utilities/` directory does not include a file named `schemaPrinter`. I've confirmed I'm using the latest version of graphql.. from`package-lock.json`:

``` 
"graphql": {
      "version": "15.0.0",
      "resolved": "https://registry.npmjs.org/graphql/-/graphql-15.0.0.tgz",
      "integrity": "sha512-ZyVO1xIF9F+4cxfkdhOJINM+51B06Friuv4M66W7HzUOeFd+vNzUn4vtswYINPi6sysjf1M2Ri/rwZALqgwbaQ==",
      "dev": true
    }
```

I figure the fix is easy enough - updating @graphql-toolkit dependencies to a newer version, but I'm moreso surprised that this issue hasn't been brought up? I can't be the only one experiencing this as your package has a ton of weekly downloads and this is a breaking issue.

**Comment by ardatan** [comment]
Fixed in 1.7.8!

**Comment by brettbullock** [comment]
Thank you!!! Really appreciate the speed in resolving here.
